config/database.yml

```diff
@@ -23,5 +23,12 @@ development:
 test:
   <<: *default
```

Comment:
Having the same config (database name in particular) for development and test will cause you problems locally. See the comments here, and see other projects' database.yml for a proper setup.